lib/IRGen/IRGenSIL.cpp

```diff
@@ -23,33 +23,35 @@
 #include "llvm/IR/IntrinsicInst.h"
 #include "llvm/IR/InlineAsm.h"
 #include "llvm/IR/Intrinsics.h"
-#include "llvm/ADT/MapVector.h"
```

Comment:
I only added the following to this section, and then ran clang-format, which reordered the entries.

```
// SWIFT_ENABLE_TENSORFLOW
#include "llvm/IR/TypeBuilder.h"
```